src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -86,8 +86,9 @@
                 <!-- or a completed Alteration? -->
                 <!-- or a completed Dissolution? -->
                 <!-- or a completed Registration? -->
+                <!-- or a limited Restoration? -->
                 <div v-else-if="filing.isCompletedIa || filing.isCompletedAlteration || filing.isCompletedDissolution ||
-                  filing.isCompletedRegistration" class="item-header__subtitle"
+                  filing.isCompletedRegistration || filing.isTypeLimitedRestoration" class="item-header__subtitle"
```

Comment:
Why wouldn't this automatically use the "else this must be a completed filing" section on line 167?

Reply:
Unfortunately, we can't use the "else" block because we need the "View / Hide details" button on lines 94 to 103.

Reply:
OK, perfect.